common/app/model/content.scala

```diff
@@ -197,15 +197,17 @@ final case class Content(
     } getOrElse (None,None)
 
     val articleMeta = if (tags.isUSMinuteSeries) {
-      Some("isMinuteArticle", JsBoolean(tags.isUSMinuteSeries))
-    } else None
+      List(
+        Some("isMinuteArticle", JsBoolean(tags.isUSMinuteSeries)),
+        Some("enhanceMobileTweets", JsBoolean(true))
```

Comment:
Hmm, yeah, I guess I was pre-empting future use of that meta for other things, but will revert this and use isMinuteArticle instead instead of trying to predict the future...